### ISSUE TITLE
Use better images?

### ISSUE THREAD

**Issue by stweil** [bertsky/mkn-kurrent-gt#2]
The PAGE XML files are currently based on page images with a reduced resolution and an additional footer (see [example](https://images.iiif.slub-dresden.de/iiif/2/data%2Fkitodo%2FNachaudeB_1761856979_1765_2%2FNachaudeB_1761856979_1765_2_tif%2Fjpegs%2F00000005.tif.medium.jpg/full/full/0/default.jpg)).

Wouldn't it be better to use page images with the original resolution and without any footer (like [this one](https://images.iiif.slub-dresden.de/iiif/2/data%2Fkitodo%2FNachaudeB_1761856979_1765_2%2FNachaudeB_1761856979_1765_2_tif%2Fjpegs%2F00000005.tif.original.jpg/full/full/0/default.jpg))?

**Comment by bertsky** [comment]
You are right – but this was not my choice. It happened when my collegues imported the data from our Presentation to your eScriptorium server. Now all the coordinates are matching the suboptimal images, and it would be difficult to recalculate them...

**Comment by stweil** [comment]
Maybe it's possible to import the better images and load the PAGE. Ideally the software should notice the mismatch of the image size and rescale the coordinates. The DFG viewer does this for ALTO. I'll run a test with eScriptorium and see if it works or if it is easy to implement. Using the alignment feature might be another possibility to match the GT texts to the better images.
